### PR TITLE
fix: wrong box-shadow when look is set to flat

### PR DESCRIPTION
**Pull request by inikolova**
Fixes https://github.com/telerik/kendo-themes/issues/2293

### CODE REVIEW

**Comment by Juveniel** [comment]
@inikolova the box-shadow part of the fix is OK. 

I have some concerns regarding changing the down arrow color. I think that we should treat this similar to [the button-icon-text](https://github.com/telerik/kendo-themes/blob/develop/tests/visual/output/material/buttons-icon-text.png) where the text and the icon have the same color. Do we have an up-to-date design for reference ?

**Comment by silviyaboteva** [comment]
+1 about the color of the arrow icon.
The box-shadow fix seems OK.

File: packages/material/scss/button/_theme.scss [hunk 1]
```diff
@@ -94,21 +94,15 @@
...
-            color: $button-text;
-        }
-
         &:disabled,
         &.k-state-disabled {
             @include box-shadow( none );
             opacity: 1;
         }
+
+        &[look="flat"] {
```
[the remark this answers is not in the document]
Comment (inikolova) [reply packages/material/scss/button/_theme.scss:103]: Currently there aren't any suitable classnames :(

**Review by Juveniel — APPROVED** [review]

**Review by silviyaboteva — APPROVED** [review]

**Comment by silviyaboteva** [comment]
@inikolova is the mentioned changes (https://github.com/telerik/kendo-angular-buttons/issues/413) have to be implemented in other suites as well?

**Comment by inikolova** [comment]
@silviyaboteva, yes - to apply this fix, **k-flat** class must be added in the other suits as well. And yes - the box-shadow of the DropDownButton is removed, however I think we could add **k-outline** class to the host element for consistency

**Comment by silviyaboteva** [comment]
@joneff this seems to be ready for merging.

**Review by joneff — APPROVED** [review]

**Pull request merged**